src/main/resources/org/jenkinsci/plugins/configfiles/ConfigFilesManagement/selectprovider.jelly

```diff
@@ -57,6 +57,11 @@ THE SOFTWARE.
 						</j:forEach>
 					</table>
 				</f:entry>
+				<f:advanced>
```

Comment:
Feedback from my change to the Credentials plugin was that this is too important to leave to an _Advanced_ block: people who need it will miss it there.